python/semantic_kernel/contents/chat_message_content_base.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft. All rights reserved.
+import sys
+from typing import TYPE_CHECKING, Any, Dict, Final
+
+from semantic_kernel.contents.const import ALL_CHAT_MESSAGE_CONTENTS, CHAT_MESSAGE_CONTENT
+
+if sys.version_info >= (3, 9):
+    from typing import Annotated
+else:
+    from typing_extensions import Annotated
+
+from xml.etree.ElementTree import Element
+
+from pydantic import Field, RootModel
+
+if TYPE_CHECKING:
+    from semantic_kernel.contents.chat_message_content import ChatMessageContent
+
+DISCRIMINATOR_FIELD: Final[str] = "type"
+
+
+class ChatMessageContentBase(RootModel):
```

Comment:
I noticed in the three methods below, there's no error handling or checking that arguments are actually supplied. Is that intended?

Reply:
yes, it relies on the "regular" pydantic validation errors if the right things are missing.